### PR TITLE
chore: test static properties

### PR DESCRIPTION
**Pull request by MarkJPerry**
## Describe your changes
Add tests for currencies

## D&D Beyond character link
N/A

## Pre-review checklist
-   [x] I have added a description of my changes
-   [x] I have formatted my code with `yarn format`
-   [x] I have added a link to a D&D Beyond character that can be used to test these changes
-   [x] I have read the [contributing guidelines](CONTRIBUTING.md)
-   [x] I agree to the [code of conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/convert.descriptors.test.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+describe('Convert DDB descriptors to Alchemy descriptors', () => {
+    test.each`
+        ddbProperty | value       | alcProperty | expected
+        ${'eyes'}   | ${'blue'}   | ${'eyes'}   | ${'blue'}
+        ${'hair'}   | ${'blonde'} | ${'hair'}   | ${'blonde'}
+        ${'height'} | ${'5ft 3"'} | ${'height'} | ${'5ft 3"'}
+        ${'skin'}   | ${'red'}    | ${'skin'}   | ${'red'}
+        ${'weight'} | ${'10'}     | ${'weight'} | ${'10'}
+    `(
+        'should covert ddbChar[$ddbProperty]:$value to $alcProperty=$expected',
```
Comment (ioveracker) [inline test/convert.descriptors.test.ts:15]:
```suggestion
        'should convert ddbChar[$ddbProperty]:$value to $alcProperty=$expected',
```

File: test/convert.currencies.test.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+describe('Convert DDB currencies to Alchemy currencies', () => {
+    test.each`
+        ddbCurrency | value | alcProperty   | expected
+        ${'cp'}     | ${1}  | ${'copper'}   | ${1}
+        ${'sp'}     | ${2}  | ${'silver'}   | ${2}
+        ${'gp'}     | ${3}  | ${'gold'}     | ${3}
+        ${'pp'}     | ${4}  | ${'platinum'} | ${4}
+        ${'ep'}     | ${5}  | ${'electrum'} | ${5}
+    `(
+        'should covert currencies[$ddbCurrency]:$value to $alcProperty=$expected',
```
Comment (ioveracker) [inline test/convert.currencies.test.ts:15]:
```suggestion
        'should convert currencies[$ddbCurrency]:$value to $alcProperty=$expected',
```

**Review by ioveracker — CHANGES_REQUESTED** [review]
Two small typos to fix up and then these are good to go! It is so nice to have this test coverage, so thank you for taking the time to go through these scenarios.

File: test/convert.currencies.test.ts (hunk 2 above)
Comment (MarkJPerry) [reply test/convert.currencies.test.ts]: Fixed

File: test/convert.descriptors.test.ts (hunk 1 above)
Comment (MarkJPerry) [reply test/convert.descriptors.test.ts]: Fixed

**Review by ioveracker — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**